network.go

```diff
@@ -628,6 +638,17 @@ func NetworkOptionInternalNetwork() NetworkOption {
 	}
 }
 
+// NetworkOptionAttachable returns an option setter to set attachable for a network
+func NetworkOptionAttachable(attachable bool) NetworkOption {
+	return func(n *network) {
+		if n.generic == nil {
+			n.generic = make(map[string]interface{})
+		}
+		n.attachable = attachable
+		n.generic[netlabel.Attachable] = attachable
```

Comment:
Why is the attachable info passed in the driver options ?
It is an information managed by libnetwork core. I don't think network drivers need to know.